RegularApi.Tests/Fixtures/ModelFactory.cs

```diff
@@ -0,0 +1,55 @@
+using System.Collections.Generic;
+using MongoDB.Bson;
+using RegularApi.Domain.Model;
+
+namespace RegularApi.Tests.Fixtures
+{
+    public static class ModelFactory
```

Comment:
I would change `DeploymentTemplateFixture` because we're fix the data always to the same, and I think that is more understandable in Tests Context

Reply:
It is not just for `DeploymentTemplate`, as it lives in the `Fixtures` namespace, I think it makes a clear context on where it belongs